service/build.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
 	implementation group: 'bio.terra.cloud-resource-lib', name: 'azure-resourcemanager-common'
 
 	// Sam
-	implementation group: "org.broadinstitute.dsde.workbench", name: "sam-client_2.12", version: "0.1-61135c7"
+	implementation group: "org.broadinstitute.dsde.workbench", name: "sam-client_2.13", version: "0.1-c53306a-SNAP"
```

Comment:
I updated the version of Sam Client because I initially was returning the policy entry memberPolicies, and it only exists in this newer version (I tried to update to the most recent few Sam commits, but that client JARs could not be found so it doesn't seem that a new JAR is generated on every commit). I am no longer returning anything that requires the updated Sam Client, but it seems good to still update the version and use more of the v2 endpoints.

Reply:
So long as our deployments aren't ahead of Sam, this seems fine to me.